ios/FluentUI.Demo/FluentUI.Demo/Demos/ListDemoController.swift

```diff
@@ -127,6 +109,16 @@ class ListDemoController: DemoController {
                                      demoControllerView.safeAreaLayoutGuide.trailingAnchor.constraint(equalTo: listView.trailingAnchor)])
     }
 
+    private func createSamplePersonaCell(cellState: MSFListCellState, samplePersonaIndex: Int) {
+        let avatar = createAvatarView(size: .medium,
+                                      name: samplePersonas[samplePersonaIndex].name,
```

Comment:
This samplePersonas being used here is the one defined in the file PersonaListViewDemoController:
https://github.com/microsoft/fluentui-apple/blob/5ddfa5e3bdf0c38e067a5d2ee6da2223cb282efc/ios/FluentUI.Demo/FluentUI.Demo/Demos/PersonaListViewDemoController.swift#L9

You can delete the code above in line 22:

```
        let samplePersonas: [PersonaData] = [
            PersonaData(name: "Kat Larrson", email: "kat.larrson@contoso.com", subtitle: "Designer", image: UIImage(named: "avatar_kat_larsson"), color: Colors.Palette.cyanBlue10.color),
            PersonaData(name: "Kristin Patterson", email: "kristin.patterson@contoso.com", subtitle: "Software Engineer", color: Colors.Palette.red10.color),
            PersonaData(name: "Ashley McCarthy", image: UIImage(named: "avatar_ashley_mccarthy"), color: Colors.Palette.magenta20.color),
            PersonaData(name: "Allan Munger", email: "allan.munger@contoso.com", subtitle: "Designer", image: UIImage(named: "avatar_allan_munger"), color: Colors.Palette.green10.color),
            PersonaData(name: "Amanda Brady", subtitle: "Program Manager", image: UIImage(named: "avatar_amanda_brady"), color: Colors.Palette.magentaPink10.color)
        ]
```